src/components/SkipButton.js

```diff
@@ -0,0 +1,13 @@
+function SkipButton({ skipTo }) {
+    return(
+        <>
+            <div className="skip-proj-btn-wrapper">
+                <a href={`#${skipTo}`} className="skip-proj-btn secondary-btn">
+                Skip Projects
+                </a>     
+            </div>          
+        </>
```

Comment:
The fragments seem redundant too, remove them since they're not doing anything.

Reply:
Sure thing. I've done the following:

- [x]: Renamed SkipButton to SkipSectionButton
- [x]: Renamed classes to skip-section-btn
- [x]: Removed fragments from SkipSectionButton